### ISSUE TITLE
Release new version

### ISSUE THREAD

**Issue by bachp** [restic/rest-server#77]
It's been a long time since the last release. There have been some improvements regarding metrics and a security fix for append-only mode.

It would be nice it these are available in a released version.

**Comment by mika** [comment]
Friendly ping, I'd also appreciate a new release of rest-server! :)

**Comment by aliron19** [comment]
I'd also appreciate a new release, please. 

Thank you.

**Comment by ar-jan** [comment]
Another vote for a new release!

**Comment by dimejo** [comment]
If you are desperately waiting for a binary with the recent commints you could aswell use the [beta releases](https://beta.restic.net/). They are automatically built for every new commit.

**Comment by mika** [comment]
@dimejo but this is about restic **rest-server**, not restic itself :) So unless I'm overlooking something, https://beta.restic.net/ doesn't provide binaries for restic rest-server?

**Comment by dimejo** [comment]
Oh! You are right, I completely missed that. Please ignore my comment ...

**Comment by traedamatic** [comment]
Hey, can I help to create a new release? We want to use the restic rest-server for our backup solution but need a static release url.

**Comment by mholt** [comment]
I've gone ahead and pushed a tag v0.9.8 to hold things over for now. I'm not sure how the release process is automated here, but that should do for now. https://github.com/restic/rest-server/releases/tag/v0.9.8

**Comment by mika** [comment]
Thanks for tagging v0.9.8, @mholt! Looks like @zcalusic did the previous release work (so we have a changelog, pre-built/official binaries,...)?

**Comment by fd0** [comment]
@mholt thanks a lot for stepping in! The release process we used before is neither documented nor automated, I'll do that for the 0.9.8 release now.

**Comment by fd0** [comment]
OK, I don't have the time right now. A few pointers:
 * The project uses restic's changelog generator [calens](https://github.com/restic/calens). For features and pull requests, a new file in `changelog/unreleased` should be added which adheres to the format and describes the change. On release, `changelog/unreleased` is renamed to `changelog/VERSION_DATE` and `calens` extracts the metadata and generates the changelog.
 * For restic, we have scripts (well, Go programs) for the release process, documented [here](https://restic.readthedocs.io/en/latest/developer_information.html#building-the-official-binaries), the scripts are in the `helpers/` directory in the restic source code.
@zcalusic What did you use to build the binaries?

**Comment by zcalusic** [comment]
A simple container, heavily inspired by your restic/builder. :)

https://github.com/zcalusic/dockerfiles/tree/1db97b84eaf48df985f1a593714829f4491aa22a/rest-server-builder

**Comment by stappersg** [comment]
28 juli 2019, happy anniversary #77

A tag release did happen,   issue #77 should be closed.

**Comment by bachp** [comment]
@stappersg It's just a tag but no binaires are provided. As #83 was closes I think this should be kept open until this is resolved.

**Comment by monkeyhybrid** [comment]
It seems this issue is down to the [lack of] automated build and release process, is that right? Would it make sense to create a separate issue for a working build system? Something that brings it into line with Restic's build process, as [@fd0 touches on](https://github.com/restic/rest-server/issues/77#issuecomment-493736981)?

**Comment by TheDreamer** [comment]
Plus, it self identifies as 0.9.7-ish.  Of course, there's no support for '-V' in prior releases...

```
$ ./rest-server -V
rest-server 0.9.7 (v0.9.8-0-ga87d968-dirty) compiled with go1.12.5 on darwin/amd64
```

**Comment by rawtaz** [comment]
We'll try to make something happen on the release front soon.

**Comment by CHTJonas** [comment]
Would be great to get the [restic/rest-server](https://hub.docker.com/r/restic/rest-server) Docker image automated at the same time as releases! You seem to have the stuff raised in this issue on your TODO lists but do comment if you're looking for someone to submit a PR 😃

**Comment by rawtaz** [comment]
Dangit. I will put it in my calendar to look into this soon. At least then I will have something to shuffle around a bit. I totally understand that everyone wants a release!

**Comment by tcardonne** [comment]
Hello! Any updates on a Docker image release for 0.9.8?

**Comment by rawtaz** [comment]
We're currently working on rebuilding the release stuff a bit, and will release a new version (0.9.9 I guess) as soon as we get a couple of small things done in the code. So I wouldn't expect a 0.9.8 update.

I presume that once we do that, you can use the current Dockerfile, or are you talking about something else than this?

**Comment by tcardonne** [comment]
Great news! I'll build my own Docker image for 0.9.8, until you release 0.9.9.
I was talking about a Docker image release on docker hub, here : https://hub.docker.com/r/restic/rest-server.

**Comment by CHTJonas** [comment]
Friendly ping! Is there anything we can do to help unblock this?

**Comment by rawtaz** [comment]
@CHTJonas Thanks, but nope. A release will be made soon (in other words, it *is* being worked on, even if it doesn't seem like it too much).

**Comment by MichaelEischer** [comment]
rest-server 0.10.0 has been released just a few hours ago: https://github.com/restic/rest-server/releases/tag/v0.10.0

**Comment by fd0** [comment]
And I hope we can do that more regularly now :)

**Comment by stappersg** [comment]
On Wed, Sep 16, 2020 at 11:27:43PM -0700, Alexander Neumann wrote:
> And I hope we can do that more regularly now :)

The good thing is that I is part of we.
